backend/de.metas.material/cockpit/src/main/sql/postgresql/ddl/views/MD_Stock_From_HUs_V.sql

```diff
@@ -34,8 +34,8 @@ FROM MD_Stock s
                  JOIN M_HU_Storage hus ON hus.M_HU_ID = hu.M_HU_ID
                  JOIN M_Locator l ON l.M_Locator_ID = hu.M_Locator_ID
                  LEFT JOIN M_Product p ON p.M_Product_ID = hus.M_Product_ID /*needed for its C_UOM_ID*/
-        WHERE hu.isactive = 'Y'
-          AND M_HU_Item_Parent_ID IS NULL
+        WHERE -- hu.isactive = 'Y' AND hu may be inactive due to a bug in material return
```

Comment:
```suggestion
        WHERE -- hu.isactive = 'Y' AND -- hu may be inactive due to a bug in material return, and technically we don't need to check for IsActive, because we have the HuStatus to check for
```